react/week3/fetchtodos/src/App.css

```diff
@@ -0,0 +1,75 @@
+body {
+  background-color: darkgrey;
+}
+h1 {
+  color: firebrick;
+}
+.App {
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  padding: 10px;
+  margin: 50px;
+}
+
+button {
+  text-align: center;
+  width: 100px;
+  height: 30px;
+  font-size: large;
+  background-color: #64fdca;
+  margin-left: 20px;
+}
+button:hover {
+  background-color: chartreuse;
+  border: 2px solid blue;
+}
+.addbutton {
```

Comment:
Your class names vary a bit in style :) 
I'd consider changing this to `.add-button` to follow the other ones